sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1494,6 +1494,91 @@ def writer(self, test_bigquery_client=None, buffer_size=None):
         buffer_size=buffer_size)
 
 
+class WriteResult:
+  def __init__(
+      self,
+      method=None,
+      destination=None,
+      schema=None,
```

Comment:
do you think we need `destination` and `schema`? Maybe we don't really need them. What are your thoughts?

Reply:
For the scope of this PR, I agree we don't need them. Maybe down the line we might find it useful to convey more information about the write configuration in the return object.

I went ahead and removed those attributes.